Week7-0527/CardsGameTS/src/Pages/layout/CardsSection.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import Card from "../../components/Card";
+import getPokemonListRandom from "../utils/getPokemonList";
+import { useMemo} from "react";
+import { useRecoilValue } from "recoil";
+import { difficultyAtom } from "../../atoms/atom";
+
+const Wrapper = styled.section`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+
+  margin: 0 1.5rem;
+  margin-bottom: 7rem;
```

Comment:
옹! 이거 그냥 
```suggestion
  margin: 0 1.5rem 7rem 1.5rem;
```
로 합쳐줘도 되겠지??